Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "Braintree", url: "https://github.com/braintree/braintree_ios", from: "5.9.0")
+        .package(name: "Braintree", url: "https://github.com/braintree/braintree_ios", from: "5.12.0")
```

Comment:
I think we normally run `swift package resolve` to update the package.resolved looking at previous PRs.

Reply:
Good catch! I forgot that our Demo app pulls in BT via SPM (hence tracked in the Package.resolved).